adafruit_rfm9x.py

```diff
@@ -848,6 +870,7 @@ def receive(
                 if fifo_length < 5:
                     packet = None
                 else:
+                    assert packet is not None
```

Comment:
Why is assert used here? If packet = None is not an error condition? Won't asset throw an error in that case?
Perhaps I just don't understand.

Reply:
I agree that this doesn't make sense.  In fact, if `packet` is `None` then the code below should fail, particularly `packet[0]`.  By the time it gets to this line, packet may in fact be `None` but I agree with @jerryneedell that in that case the `if` statement would catch it.

It may be because mypy doesn't know that `self._read_u8()` can only return a positive number.  A negative value would conceivably fall through the cracks but that would never happen.

Reply:
Right on, as I said, this is just a little trick to get mypy to be quiet about these sorts of things. I'll remove it.

Reply:
Since I don't know what mypy is, I'll defer to the other librarians on this.  At first I thought it might cause an unexpected error but that does not appear to be the case. We have worked very hard to remove unnecessary code in this library to keep it small enough for the M0 boards. I'd support removing it if it is not necessary.

Reply:
Sweet!  This is on my list to review!